src/pages/commitqueue/CommitQueueCard.tsx

```diff
@@ -78,6 +94,7 @@ export const CommitQueueCard: React.FC<Props> = ({
             </Container>
           </CommitInfo>
         ) : (
+          // valid data should not get to here anymore, but this is left as a sanity check
```

Comment:
If we don't expect to hit this conditional its better to remove it To prevent it from being forgotten.

Reply:
I realized I forgot a corner case so I updated this comment